dbms/src/test/java/org/polypheny/db/constraints/ForeignKeyConstraintTest.java

```diff
@@ -445,16 +437,18 @@ public void testUpdateInNoConflict() throws SQLException {
     }
 
 
-    @Test
-    public void testUpdateInConflict() throws SQLException {
+    @ParameterizedTest(name = "{index}. Create Index: {0}")
+    @ValueSource(booleans = { false, true })
+    @Tag("cottontailExcluded") // only with indexes
```

Comment:
Why is this cottontail exclude necessary?

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520963686